package/boot/uboot-sunxi/patches/130-fix-mkimage-host-build.patch

```diff
@@ -0,0 +1,24 @@
+--- a/tools/image-host.c
++++ b/tools/image-host.c
+@@ -1125,6 +1125,7 @@ static int fit_config_add_verification_d
+  * 2) get public key (X509_get_pubkey)
+  * 3) provide der format (d2i_RSAPublicKey)
+  */
++#ifdef CONFIG_TOOLS_LIBCRYPTO
```

Comment:
Please add a description to the patch.
Are you adding this to work around a compile error you are getting with libressl?
Please post the compile error you get.